packages/medusa/src/services/swap.ts

```diff
@@ -975,7 +975,8 @@ class SwapService extends TransactionBaseService {
             item_id: i.id,
             quantity: i.quantity,
           })),
-          { swap_id: swapId, metadata }
+          { swap_id: swapId, metadata },
+          { locationId: config.location_id }
```

Comment:
thought (non-blocking): somewhat unrelated to this PR but feel that these two objects should be merged into a common `options` or `context`. Doesn't have to go into this PR, but just thought I'd point out that it looks awkward.